e2e/bitrise.yml

```diff
@@ -825,6 +870,50 @@ workflows:
     - _run
     - _check_outputs
 
+  test_relaunch_tests_for_each_repetition_succeeds:
+    before_run:
+    - _expose_xcode_version
+    steps:
+    - script:
+        inputs:
+        - content: |-
+            #!/bin/env bash
+            set -e
+            if [[ ${XCODE_MAJOR_VERSION} -lt 13 ]]; then
+              echo "This test case requires Xcode >= 13, skipping..."
+              exit 0
+            fi
+            envman add --key XCODE_MAJOR_VERSION_AT_LEAST_13 --value "true"
+    - bitrise-run:
+        run_if: '{{enveq "XCODE_MAJOR_VERSION_AT_LEAST_13" "true"}}'
+        inputs:
+        - workflow_id: utility_test_relaunch_tests_for_each_repetition_succeeds
+        - bitrise_config_path: ./e2e/bitrise.yml
+
+  utility_test_relaunch_tests_for_each_repetition_succeeds:
+    envs:
+    - TEST_APP_NUMBER_OF_INITIAL_TEST_SUCCESSES: 1
+    - TEST_APP_URL: https://github.com/bitrise-io/sample-swift-project-with-parallel-ui-test.git
+    - TEST_APP_BRANCH: STEP-1205-eventually-failing-in-memory-tests
```

Comment:
Update this here to `master` if this PR is merged: https://github.com/bitrise-io/sample-swift-project-with-parallel-ui-test/pull/8.